docs/performance/tune-performance.md

```diff
@@ -170,6 +170,7 @@ Currently, there are no special provisions to employ mimalloc on Linux. It is re
 ### Thread management
 
 * Use the appropriate ORT API to set intra and inter op num threads. Inter op num threads is only used when parallel execution is enabled.
+* In most cases, by default, both of these values are set to the number of physical cores. The exception is on posix mobile platforms, where the default is determined by `std::max(1, static_cast<int>(std::thread::hardware_concurrency() / 2))`
```

Comment:
>The exception is on posix mobile platforms, where the default is determined by `std::max(1, static_cast<int>(std::thread::hardware_concurrency() / 2))` [](http://example.com/codeflow?start=91&length=151)

This is fine.
But I doubt if we should go into such as detail pasting c++ code. Note one thing - even with "std::max(...)" ort is simply trying to estimate the number of physical cores of the (mobile) system.
So simply just keeping:
"In most cases, by default, both of these values are set to the number of physical cores".
Should suffice.

Reply:
Another thing worth mentioning is:

Ort by default will attach every created thread to a certain core to minimize cache miss overhead; However, with customized number of thread, ort will not attach any thread to any core, this might introduce notable performance variance.